network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -170,7 +171,10 @@ class BuyersGuideProductCategory(index.Indexed, TranslatableMixin, LocalizedSnip
     )
 
     panels = [
-        FieldPanel('name'),
+        FieldPanel(
```

Comment:
Wanted to note that the limit for this field was already set to 100, so I just had to add the widget